fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalHashJoin.java

```diff
@@ -88,6 +88,16 @@ public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
 
     @Override
     public String toString() {
+        if (statsDeriveResult != null) {
+            return Utils.toSqlString("PhysicalHashJoin",
+                    "type", joinType,
+                    "hashJoinCondition", hashJoinConjuncts,
+                    "otherJoinCondition", otherJoinCondition,
+                    "estimateRows", statsDeriveResult.getRowCount(),
+                    "isReduced", statsDeriveResult.isReduced,
+                    "width", statsDeriveResult.width
```

Comment:
why print stats info in plans? i don't think this is a good idea

Reply:
it help us to check whether the plan is as good as we expected.